apps/web/src/app/[locale]/_components/hero.tsx

```diff
@@ -184,28 +185,44 @@ export async function Hero() {
             <Balancer>{t('description')}</Balancer>
           </p>
           <div className="flex flex-col-reverse gap-3 md:flex-row">
-            <Button
-              asChild
-              className="hero-join-button group relative mx-auto w-fit overflow-hidden rounded-xl p-[2px] font-bold transition-all duration-300 hover:bg-transparent hover:shadow-[0_0_2rem_-0.5rem_#FFB800] dark:hidden md:mr-0 lg:mr-auto"
-            >
-              <Link href="/waitlist">
-                <span className="inline-flex h-full w-fit items-center gap-1 rounded-[10px] bg-white px-4 py-2 text-[#FFB800] transition-all duration-300">
-                  <Mail className="mr-1 h-4 w-4 stroke-[3]" />
-                  {t('buttons.waitlist')}
-                </span>
-              </Link>
-            </Button>
-            <Button
-              asChild
-              className="hero-join-button-dark group relative mx-auto hidden w-fit overflow-hidden rounded-xl p-[1px] font-bold transition-all duration-300 dark:block dark:hover:shadow-[0_0_2rem_-0.5rem_#fff8] md:mr-0 lg:mr-auto"
-            >
-              <Link href="/waitlist">
-                <span className="inline-flex h-full w-fit items-center gap-1 rounded-xl px-4 py-2 transition-all duration-300 dark:bg-neutral-900 dark:text-white group-hover:dark:bg-black">
-                  <Mail className="mr-1 h-4 w-4 stroke-[3]" />
-                  {t('buttons.waitlist')}
-                </span>
-              </Link>
-            </Button>
+            {!session && (
+              <Button
+                asChild
+                className="hero-join-button group relative mx-auto w-fit overflow-hidden rounded-xl p-[2px] font-bold transition-all duration-300 hover:bg-transparent hover:shadow-[0_0_2rem_-0.5rem_#FFB800] dark:hidden md:mr-0 lg:mr-auto"
+              >
+                <Link href="/waitlist">
+                  <span className="inline-flex h-full w-fit items-center gap-1 rounded-[10px] bg-white px-4 py-2 text-[#FFB800] transition-all duration-300">
```

Comment:
whats the diff between these two? can one be removed or consolidated?

Reply:
There are two buttons, one for light theme animation and one for dark theme animation.